clang/test/Analysis/issue-70464.cpp

```diff
@@ -66,3 +66,23 @@ struct Derived : Base {
 void entry() { Derived test; }
 
 } // namespace delegate_ctor_call
+
+// Additional test case from issue #59493
+namespace init_list_array {
+
+struct Base {
+  int foox[1];
```

Comment:
I'd suggest a more realistic array size for the test and also test some of the subsequent elements, including zero initialized elements.